### PR TITLE
add restore flag

### PR DESCRIPTION
**Pull request by yuuu**
I like your tool very much. But you also know that there is a problem with this tool "Once you start dragon extermination, you can not retire".

So I added the `-r` option. With this option you can restore to the previous label. 

I am pleased if you can merge.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -48,15 +54,20 @@ func main() {
...
 		opt.DryRun = c.Bool("dry-run")
+		opt.Restore = (c.Bool("restore") || c.Bool("r"))
 		run(opt)
 		return nil
 	}
 
 	app.Flags = []cli.Flag{
 		cli.BoolFlag{
-			Name:  "dry-run",
+			Name:  "dry-run, d",
```
Comment (k-masatany) [inline main.go]: Since `-d` is associated with delete, we do not want to apply this.

File: main.go [hunk 2]
```diff
@@ -131,3 +164,42 @@ func confirm(message string) string {
...
+	for _, label := range labels {
+		writer.WriteString(fmt.Sprintf("%s, %s\n", *label.Name, *label.Color))
+	}
+	writer.Flush()
+
+	return nil
+}
+
+// restore labelset
+func restoreLbelset(path string) ([]map[string]string, error) {
```
Comment (k-masatany) [inline main.go]: Please Fix typo, restoreLbelset -> restoreLabelset.

File: main.go (hunk 1 above)
Comment (yuuu) [reply main.go]: I restored it.

File: main.go (hunk 2 above)
Comment (yuuu) [reply main.go]: Thank you for pointing out typo. Fixed.

**Pull request merged**